README.md

```diff
@@ -1,328 +1,526 @@
 
-Send data into Slack using this GitHub Action!
+## Example workflows
 
-## Example Workflows
+For examples on how to leverage this Action in workflows, check out
+[example workflows we have][examples] available.
 
-For examples on how to leverage this in your workflows, check out the [example workflows we have](https://github.com/slackapi/slack-github-action/tree/main/example-workflows).
+## Sending variables
 
-## Sending Variables
+There are different [techniques to send data](#sending-techniques) into Slack
+and whichever one is chosen will require a certain set of customized inputs, as
+described later.
 
-You can provide data to send with this GitHub Action from various sources:
+You can provide data to send to Slack from this GitHub Action and either source:
 
-* The default event [context][event-context] with a [payload][event-payload] matching the GitHub event.
-* A custom JSON payload with optional [variables][variables] provided in the GitHub Action step.
+- The default event [context][event-context] with a [payload][event-payload]
+  matching the GitHub event.
+- A custom payload with optional [variables][variables] provided in the GitHub
+  Action step.
 
-The custom payload can be provided inline in your GitHub workflow or from a file, [detailed with technique 1](#technique-1-slack-workflow-builder), and will be used instead of the default event context if included.
+These input options are valid for all techniques, but some techniques require
+specific constraints with certain requirements for valid inputs.
 
-## How to Send Data to Slack
+Additional [configurations](#additional-configurations) and other details are
+also available for more customizations to the provided payload.
 
-This package has three different techniques to send data to Slack:
+## Sending techniques
 
-1) Send data to Slack's Workflow Builder (requires a paid Slack instance).
-2) Send data via a Slack app to post to a specific channel (use an existing custom app or create a new one).
-3) Send data via a Slack Incoming Webhook URL (use an existing custom app or create a new one).
+This Action offers three different techniques to send data to Slack:
 
-The recommended way to use this action is with Slack's Workflow Builder (if you're on a paid Slack plan).
+1. [**Technique 1**](#technique-1-slack-workflow-builder): Send data with a
+   webhook to start a workflow in [Workflow Builder][wfb].
+2. [**Technique 2**](#technique-2-slack-api-method): Send data using
+   [a Slack API method][methods] and a secret token with required scopes.
+3. [**Technique 3**](#technique-3-slack-incoming-webhook): Send data as a
+   message with a Slack [incoming webhook][incoming-webhook] URL.
 
 ### Technique 1: Slack Workflow Builder
 
-> ❗️ This approach requires a paid Slack plan; it also doesn't support any text formatting
+> :memo: This technique requires [a Slack paid plan][plans] to use Workflow
```

Comment:
🤔 Hmm is this pricing change instead related to "premium" workflows within Workflow Builder? Hidden in the [WFB FAQ](https://slack.com/features/workflow-automation#:~:text=Frequently%20asked%20questions) is a note that omits free plans from combining steps which I'm leaning on as a source of truth for the moment.

But please let me know if you think this still seems off since I'm open to exploring further! Otherwise I'll mark this as resolved, but will also keep an eye out for future changes to pricings and plans 👁️‍🗨️

Reply:
Ah no you are 100% correct, my bad! Resolving this comment.